app/src/main/res/layout/error_panel.xml

```diff
@@ -72,4 +72,19 @@
         android:visibility="gone"
         tools:visibility="visible" />
 
+    <Button
+        android:id="@+id/error_open_in_browser"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="-4dp"
```

Comment:
Remove `layout_marginBottom` from `error_retry_button` and remove this line: negative margins are not a good idea, and also at the moment the buttons are not equally spaced out.